### ISSUE TITLE
Icon Request: "Untagged" or "Not For Sale"

### ISSUE THREAD

**Issue by onkoe** [google/material-design-icons#1743]
I'd like to see an adapted version of the ["Sell"](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:flash_off:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=flash&icon.size=24&icon.color=%235f6368) glyph that has a slash through it. The normal variant means "tagged" within my app, but I'd love to see an "untagged variant. 

Here's a draft based on the ["Flash Off"](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:sell:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=sell&icon.size=24&icon.color=%235f6368) glyph:

<img width="967" alt="image" src="https://github.com/user-attachments/assets/abb9f4ea-37ba-4831-a2c9-f06428db87da">

It kinda looks like a funky pencil up close, but it's nice when you zoom out. There are similar icons from other collections, like Pictogrammers/MDI's "tag-off-outline". However, these resources are not as universal as Material Design itself, making them less useful to folks like me.

Please reach out if any additional details are needed. Thank you!

**Comment by tphinney** [comment]
All slash-through icons in this set have the slash the other way, but yes, something like that would be possible—perhaps just rotated 90° clockwise.

**Comment by onkoe** [comment]
Thanks! The point you made is considerable, though I can't seem to find a matching ideal in the "Designing Icons" documentation.

Is it more of a convention? 

Either way, it might be necessary to consider additional variants of the tag icon which are fully vertical or horizontal to better accommodate the slash. Would you like some drafts/examples of the tag with additional designs, or do you feel comfortable as-is?

**Comment by tphinney** [comment]
Yes, it’s just a convention, but it’s consistent in all the existing icons
so it is pretty set.

No more info needed I don’t think.

Thanks!
